### DISCUSSION TITLE
v2.6.0-rc2

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#51] [Announcements]
## What's Changed
* feat: add hash generation multi-threading support, change `--rebuild-hash-cache` value type from bool to int (number of CPU cores)
* fix: prevent inference tensor version counter tracking issue for GroundingDINO after using Enhance (discussion in https://github.com/lllyasviel/Fooocus/discussions/3213)

**Full Changelog**: https://github.com/mashb1t/Fooocus/compare/v2.6.0-rc1...v2.6.0-rc2

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.6.0-rc2'>v2.6.0-rc2</a>.</em>

**Comment by mykeehu** [comment]
The update does not run completely, I get errors like this when updating a Windows package:
```
Version mismatch for translators: Installed version 5.8.9 does not meet requirement translators==5.9.2
Installing requirements
Couldn't install requirements.
Command: "I:\Fooocus\python_embeded\python.exe" -m pip install -r "requirements_versions.txt" --prefer-binary
Error code: 1
.
.
.
.

ERROR: Could not install packages due to an OSError: [WinError 5] A hozzáférés megtagadva: 'I:\\Fooocus\\python_embeded\\Lib\\site-packages\\cv2\\cv2.pyd'
Consider using the `--user` option or check the permissions.


[notice] A new release of pip is available: 23.2.1 -> 24.1.2
[notice] To update, run: I:\Fooocus\python_embeded\python.exe -m pip install --upgrade pip

CMD Failed requirements: install -r "requirements_versions.txt"
```

How can I open venv to manually install/update it?

**Reply by mashb1t** [reply]
you can manually install/update the packages by executing this command in the Fooocus folder (the one with the config.txt):
`..\python_embeded\python.exe -m pip install -r .\requirements_versions.txt`

**Comment by mykeehu** [comment]
Thank you, now you've done it!
